src/Mod/Spreadsheet/Gui/SheetModel.cpp

```diff
@@ -76,6 +76,22 @@ int SheetModel::columnCount(const QModelIndex &parent) const
 }
 
 
+QVariant format_cell_display(QString value, const Cell * cell)
```

Comment:
Use camel casing `formatCellDisplay`.

Why is that function introduced as a global function rather than a member one ?

Reply:
i don't know, i usually don't use member, being a C coder xD

I was seeing that more as an helper function, so not really a functionality of the class

Reply:
I like it as a non-member, with two caveats (neither of which is a deal-breaker, and could be addressed in a later cleanup PR):
1) we don't use snake-case in our C++, as a general rule, and
2) it should be in an anonymous namespace to avoid potential linker errors down the road.

Reply:
1. ok format_cell_display changed to formatCellDisplay
2. I's just putting my function inside a namespace
{
...
}
block, is it ?

Reply:
Yes, exactly right.